### DISCUSSION TITLE
Alerting at Kafka-UI

### DISCUSSION THREAD

**Discussion by RomitMehta** [provectus/kafka-ui#2362] [General]
How cool it would be if it had an alerting option to alert based on some configurable parameters?
Eg: In-sync replicas > 0 or consumer lag at a point of time

**Comment by Haarolean** [comment]
Hey, we have it planned: #2245

**Reply by ankitknit4** [reply]
Any updates on this feature ?

**Reply by Haarolean** [reply]
@ankitknit4 please take a look at our roadmap
https://github.com/provectus/kafka-ui/projects/8
